packages/inspector/src/components/EntityInspector/EntityHeader/EntityHeader.tsx

```diff
@@ -72,6 +72,9 @@ export default React.memo(
 
     const hasGltfContainer = useHasComponent(entity, sdk.components.GltfContainer);
     const hasMeshCollider = useHasComponent(entity, sdk.components.MeshCollider);
+    const isGltfContainerOnEntity = sdk.components.GltfContainer.has(entity);
```

Comment:
wdyt about renaming from `isGltfContainerOnEntity` to `hasGltfContainerOnEntity`? it makes more semantic sense

```suggestion
    const hasGltfContainerOnEntity = sdk.components.GltfContainer.has(entity);
```